tests/PySys/mqtt_port/mqtt_port_set/pysystest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pysystest type="auto">
+	<description>
+		<title>
+			Validate the mqtt port set
+			Set the mqtt port using tedge command and verify the 
+			port has set properly or not.
```

Comment:
I am not sure what is going on here with those descriptions but when you run the tests it prints it precisely as it's written here, with tabs and multiple lines. Maybe we should use some escape characters?

Reply:
I would rather propose a one line title very similar to the folder name. I think it is really supposed to be a title here.